src/components/bcros/filing/CommonTemplate.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div :data-cy="`filingHistoryItem-${dataCy}`" class="w-full bg-white p-3 rounded-sm">
+    <div data-cy="filingHistoryItem-header" class="flex flex-col">
+      <h3 class="item-header-title">
+        <slot name="title">
+          <!-- todo: should we internationalize this, using lang file ??? -->
```

Comment:
My instinct on this is yes, beacuse it would be consistent with the below translation of filing detail